### PR TITLE
README.md : HyperLinking the encore image.

### PR DESCRIPTION
**Pull request by Abubakrce19**
I hyperlinked the encore image with its encore's official website, so a user can access website with single click,
Minor but impactful.
Thank you!

### CODE REVIEW

**Comment by Abubakrce19** [comment]
@eandre

**Comment by Abubakrce19** [comment]
@eandre

File: README.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
-  <img width="1500px" src="https://encore.dev/assets//branding/github_header.png" alt="Encore - Backend development engine">
+  <a href="https://encore.dev">
+<img width="1500px" src="https://encore.dev/assets//branding/github_header.png" alt="Encore - Backend development engine">
+</a>
```
Comment (DomBlack) [inline README.md:3]: Do you mind fixing the spacing?

```suggestion
  <a href="https://encore.dev">
    <img width="1500px" src="https://encore.dev/assets//branding/github_header.png" alt="Encore - Backend development engine">
  </a>
```

**Review by DomBlack — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Abubakrce19) [reply README.md]: > Do you mind fixing the spacing?
@DomBlack 
Done!!
Sorry for that!

**Pull request merged**